apps/src/p5lab/spritelab/blocks.js

```diff
@@ -501,19 +501,27 @@ export default {
       },
       removeVar: Blockly.Blocks.variables_get.removeVar,
     };
-    generator.sprite_variables_get = function () {
```

Comment:
out of curiosity: was this block broken as-is or was this just a clean up to make things easier in the future?

Reply:
Just clean up!